resources/site/changelog.md

```diff
@@ -5,6 +5,12 @@
 
 # Changelog
 
+# [0.0.0-beta.33](https://github.com/SchweizerischeBundesbahnen/scion-workbench/compare/0.0.0-beta.32...0.0.0-beta.33) (2020-02-21)
+
+### Features
+
+* add support Angular 9, drop support for Angular < 9 [#197](https://github.com/SchweizerischeBundesbahnen/scion-workbench/issues/197)
```

Comment:
The commit message should be prefixed with `chore: ` instead of `add: `. Please correct your commit message and generate the changelog anew.

`chore: add support for angular 9, drop support for angular < 9`

Reply:
Done